package.json

```diff
@@ -29,5 +29,9 @@
     "preupdate-vendor": "mv vendor/.gitignore .vendor-gitignore",
     "postupdate-vendor": "mv .vendor-gitignore vendor/.gitignore",
     "update-vendor": "yarn install --prod --flat --frozen-lockfile --modules-folder=vendor"
+  },
+  "volta": {
+    "node": "16.15.0",
+    "yarn": "1.19.1"
```

Comment:
these are the current versions used in CI and adding it here makes dev easier for anyone who uses volta